tasks/task2.js

```diff
@@ -0,0 +1,44 @@
+import fs from 'fs';
+import csv from 'csvtojson';
+
+const inputFileURL = './csv/nodejs-hw1-ex1.csv';
+const outputFileURL = 'nodejs-hw1-ex1.txt';
+
+const readable = fs.createReadStream(inputFileURL, 'utf8');
+const writable = fs.createWriteStream(outputFileURL);
+let lineCount = 0;
+
+const onError = (err, module = 'parsing') => {
```

Comment:
function is far away from usage. More verbose name (onCsvParseError) would be helpful